src/styles/blocks.scss/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` property is usually applied to all elements in the project. Consider moving it to the general stylesheet.